services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+from django.db.models import DateTimeField
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: DateTimeField,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie=movie,
+                                cinema_hall=cinema_hall)
+
+
+def get_movies_sessions(
+        session_date: Optional[DateTimeField] = None
```

Comment:
The parameter `session_date` should be of type `datetime.date`, not `DateTimeField`. `DateTimeField` is a Django model field type, whereas `datetime.date` is the appropriate type for a date value.